subsys/net/lib/aws_jobs/src/aws_jobs.c

```diff
@@ -428,6 +428,7 @@ int aws_jobs_unsubscribe_expected_topics(struct mqtt_client *const client,
  */
 static int construct_job_id_get_topic(const u8_t *client_id,
 				      const u8_t *job_id,
+				      const u8_t *suffix,
```

Comment:
Missing docs for this argument

Reply:
Added

Reply:
* @param[in] suffix Suffix at the end of the constructed mqtt topic. The topic
 *	does not include a forward slash `/` so this has to be included when
 *	passing a suffix to the topic.